src/dcap_provider.cpp

```diff
@@ -234,15 +234,15 @@ bool get_cache_expiration_time(const string &cache_control, const string &url, t
                 return false;
             }
         }
-        catch (std::invalid_argument e)
+        catch (std::invalid_argument& e)

```

Comment:
[nit]:  `std::invalid_argument&` -> `const std::invalid_argument&`

Reply:
changed to const reference. It is a better practice